foosball/users/models.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+from django.contrib.auth.models import AbstractUser
+from django.core.urlresolvers import reverse
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+
+
+@python_2_unicode_compatible
+class User(AbstractUser):
+
+    # First Name and Last Name do not cover name patterns
+    # around the globe.
+    name = models.CharField(_("Name of User"), blank=True, max_length=255)
```

Comment:
I don't think we need this name field for users. Of course this can be changed later.